app/assets/stylesheets/stream_element.css.scss

```diff
@@ -12,7 +9,42 @@
   }
 }
 
+#main_stream > div > .photo {
+  & > .media {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 4<br>Unnecessary parent selector (&)